libs/langchain/langchain/utilities/gitlab.py

```diff
@@ -0,0 +1,311 @@
+"""Util that calls gitlab."""
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
+
+from langchain.pydantic_v1 import BaseModel, Extra, root_validator
+from langchain.utils import get_from_dict_or_env
+
+if TYPE_CHECKING:
+    from gitlab.v4.objects import Issue
+
+
+class GitLabAPIWrapper(BaseModel):
+    """Wrapper for GitLab API."""
+
+    gitlab: Any  #: :meta private:
+    gitlab_repo_instance: Any  #: :meta private:
+    gitlab_repository: Optional[str] = None
```

Comment:
could we add docstrings for these params? otherwise everything looks great!

Reply:
I added docstrings for the public params. Is this fine?